masterlist.yaml

```
@@ -2113,6 +2113,35 @@ plugins:
             text: 'Приставката е стара - изисква се само ако менюто за конфигуриране на модификациите не работи.'
         condition: 'file("Mart''s Mutant Mod - Master Menu Module.esp") or checksum("Mart''s Mutant Mod.esm", DC2E29E4)'
 
+  # Raiders Redone and Modules #
+  - name: '(RR(Females|GoreGrafitti)Removed|RaidersRedoneModular01)\.esp'
```

Removed the header comment as it isn't that useful for this mod.

I would place the main filename first in a regex when possible.
It's easier to find when scrolling through code,
```suggestion
  - name: '(RaidersRedoneModular01|RR(Females|GoreGrafitti)Removed)\.esp'
```